plugin/rewrite/README.md

```diff
@@ -222,6 +222,44 @@ Note that the above syntax is strict.  For response rewrites, only `name`
 rules are allowed to match the question section. The answer rewrite must be
 after the name, as in the syntax example.
 
+#### Explicit Response Value Rewrite
```

Comment:
Thanks for the PR.

```suggestion
##### Example: PTR Response Value Rewrite
```

FWIW, I also happen to find the existing use of "Explicit ..." in other subtitles to indicate these are examples of the previous section to be confusing.